src/game/lara/lara_hair.c

```diff
@@ -48,9 +50,10 @@ void Lara_Hair_Initialise(void)
 void Lara_Hair_SetLaraType(GAME_OBJECT_ID lara_type)
 {
     m_LaraType = lara_type;
+    m_InCutscene = m_LaraType != O_LARA;
```

Comment:
After seeing it in action, let's turn it into a normal variable within `Lara_Hair_Control`. We should keep as little state global as possible.